js/addToTask.js

```diff
@@ -0,0 +1,74 @@
+import {
+  createTaskButton,
+  content,
+  list,
+  inputTask,
+  input,
+  addTask,
+  trashInput,
+  today,
+  COMPLETE,
+  EDIT,
+  DELET,
+  CLICK_EVENT,
+  HIDE,
+} from "/js/constants.js"; //named import
+
+export let LIST = [];
+export let id = 0;
+let tarikh;
+function addToDo(toDo, id, done, edit, trash) {
+  if (trash) {
+    return;
+  }
+
+  //Presenting Todays date in task list inside li tag
+  const options = {
+    day: "numeric",
+    month: "numeric",
+    year: "numeric",
+  };
+
+  tarikh = today.toLocaleDateString("en-us", options);
+  const item = `<li class="item">                  
+                  <p class="text">${toDo}</p>
+                  <!-- data presenter div tag with date class  -->
+                  <div class = "date" id = "${id}"> Created At:  ${tarikh} </div>
+                  <img src= "icons/done.svg" job = "${COMPLETE}" id ="${id}"/>
+                  <img src = "icons/edit.svg" job = "${EDIT}" id = "${id}"/>
+                  <img src = "icons/delete.svg" job="${DELET}" id="${id}"/>
```

Comment:
What is the purpose of job attributes here? Is that a custom attribute, or?

Reply:
@hhu-wtag Bhai Is using custom attributes such as `job` in HTML considered good practice?

Reply:
Normally, we can target a HTML element using `id` as `id` is supposed to be unique in the whole document. So we can omit the custom attribute here and continue using `id`. Sometimes we may need custom attributes. We can use `data-*` attributes then.